packages/rcbc/test.R

```diff
@@ -0,0 +1,3 @@
+options(download.file.method='curl')
+install.packages('remotes', repos='https://cran.rstudio.com')
+remotes::install_github("dirkschumacher/rcbc")
```

Comment:
test that something more than installing the package works.

Reply:
I've added a test from their README.

Reply:
at https://github.com/rstudio/shinyapps-package-dependencies/pull/210/commits/24a5cfc84ff9790af416c5cde6cbe38400a16f83#diff-6e32d8cd5aa012dd4d8552761ab931e4R4